docs/source/security.rst

```diff
@@ -0,0 +1,38 @@
+.. _security_advises:
```

Comment:
To have this included in toc tree, there is a need to add a reference to the index - https://github.com/thoth-station/adviser/blob/master/docs/source/index.rst